recipes/r-colorramps/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = '2.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-colorramps
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: colorRamps_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/colorRamps_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/colorRamps/colorRamps_{{ version }}.tar.gz
+  sha256: 3f10eab0602a42a2e5a79c7c04638be06cbf1e6f23979a1bd95d853e10940b6a
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('colorRamps')"  # [not win]
+    - "\"%R%\" -e \"library('colorRamps')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=colorRamps
+  license: GPL
```

Comment:
Do you know the version?

Reply:
The [DESCRIPTION file](https://cran.r-project.org/web/packages/colorRamps/DESCRIPTION) says only "GPL". The [CRAN entry](https://cran.r-project.org/package=colorRamps) expanded it to say:

> License: GPL-2 | GPL-3 [expanded from: GPL]

Would you prefer I put GPL2 or GPL3?

Reply:
That is up to the maintainers, conda-forge does not have a policy when there are alternatives.